logistics/scripts/modules/settings/settings-management.psm1

```diff
@@ -0,0 +1,351 @@
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+& "$PSScriptRoot\..\..\..\..\logistics\scripts\modules\load-path-resolver.ps1"
+Import-Module -Force -Scope Global (Get-RepositoryResolvedPath 'logistics\scripts\modules\utility\hashtable.psm1')
+Import-Module -Force -Scope Global (Get-RepositoryResolvedPath 'logistics\scripts\modules\config\config-management.psm1')
+
+function Get-DevelopmentSettingsByProject {
+    return @{
+        "EdFi.Ods.WebApi.NetCore"             = @{
+            Urls              = "http://localhost:54746"
+            ApiSettings       = @{
+                Engine = ""
+            }
+            ConnectionStrings = @{ }
+            Plugin            = @{
+                Scripts = @("development")
+            }
+            Logging           = @{
+                LogLevel = @{
+                    Default = "Debug"
+                }
+            }
+        }
+        "EdFi.Ods.Api.IntegrationTestHarness" = @{
+            Urls              = "http://localhost:8765"
+            ApiSettings       = @{
+                Engine = ""
+            }
+            ConnectionStrings = @{ }
+        }
+        "EdFi.Ods.SandboxAdmin.Web"           = @{
+            Urls              = "http://localhost:38928"
+            Engine            = ""
+            ConnectionStrings = @{ }
+            Logging           = @{
+                LogLevel = @{
+                    Default = "Debug"
+                }
+            }
+        }
+        "EdFi.Ods.SwaggerUI"                  = @{
+            Urls             = "http://localhost:56641"
+            WebApiVersionUrl = "http://localhost:54746"
+            Logging          = @{
+                LogLevel = @{
+                    Default = "Debug"
+                }
+            }
+        }
+    }
+}
+
+function Get-CredentialSettingsByProject {
+
+    function Get-RandomString([int] $length = 20) {
+        return ([char[]]([char]65..[char]90) + ([char[]]([char]97..[char]122)) + 0..9 | Sort-Object { Get-Random })[0..$length] -join ''
+    }
+
+    $populatedKey = Get-RandomString
+    $populatedSecret = Get-RandomString
+
+    return @{
+        "EdFi.Ods.SandboxAdmin.Web" = @{
+            User = @{
+                "Test Admin" = @{
+                    Email             = "test@ed-fi.org"
+                    Password          = Get-RandomString
+                    Admin             = "true"
+                    NamespacePrefixes = @(
+                        "uri://ed-fi.org"
+                        "uri://gbisd.org"
+                    )
+                    Sandboxes         = @{
+                        "Populated Demonstration Sandbox" = @{
+                            Key     = $populatedKey
+                            Secret  = $populatedSecret
+                            Type    = "Sample"
+                            Refresh = "true"
```

Comment:
We actually shouldn't be setting these up as refresh, since this is a local environment, but I'm guessing this was what the original code was doing?  This will queue up a hangfire job to reset it nightly.

Reply:
This was how the original code was doing it but I agree so I modified them to be false.